client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -144,46 +158,57 @@ class EditorConfirmationSidebar extends Component {
 		const isSidebarActive = this.props.status === 'open';
 		const isOverlayActive = this.props.status !== 'closed';
 
-		return (
-			<div
-				className={ classnames( {
-					'editor-confirmation-sidebar': true,
-					'is-active': isOverlayActive,
-				} ) }
-			>
-				{ this.renderPublishingBusyButton() }
+		if ( ! this.state.hasUnclosed ) {
+			return (
+				<div className="editor-confirmation-sidebar">
+					<div key="sidebar" className="editor-confirmation-sidebar__sidebar" />
+				</div>
+			);
+		}
 
+		return (
+			this.state.hasUnclosed && (
 				<div
 					className={ classnames( {
-						'editor-confirmation-sidebar__sidebar': true,
-						'is-active': isSidebarActive,
+						'editor-confirmation-sidebar': true,
+						'is-active': isOverlayActive,
 					} ) }
 				>
-					<div className="editor-confirmation-sidebar__ground-control">
-						<div className="editor-confirmation-sidebar__close">
-							<Button
-								borderless
-								onClick={ this.getCloseOverlayHandler( 'dismiss_x' ) }
-								title={ this.props.translate( 'Close sidebar' ) }
-								aria-label={ this.props.translate( 'Close sidebar' ) }
-							>
-								<Gridicon icon="cross" />
-							</Button>
-						</div>
-						<div className="editor-confirmation-sidebar__action">
-							{ this.renderPublishButton() }
+					{ this.renderPublishingBusyButton() }
+
+					<div
+						key="sidebar"
+						className={ classnames( {
+							'editor-confirmation-sidebar__sidebar': true,
+							'is-active': isSidebarActive,
+						} ) }
```

Comment:
The same `classnames` call improvement can be done here.

Reply:
👍